mrp_bom_dismantling/models/mrp_bom.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# © 2016 Cyril Gaudin (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import _, api, exceptions, fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = 'mrp.bom'
+
+    dismantling = fields.Boolean(string='Dismantling', default=False)
```

Comment:
field's optional parameter(string & default) does not have significance as the value is same as not provided